projects/aws/bottlerocket-bootstrap/pkg/kubeadm/controlplane_init.go

```diff
@@ -124,9 +126,29 @@ func controlPlaneInit() error {
 		return errors.Wrap(err, "Error waiting for kubelet to come up")
 	}
 
+	cmd = exec.Command(kubeadmBinary, utils.LogVerbosity, "version", "-o=short")
+	out, err = cmd.CombinedOutput()
+	if err != nil {
+		return errors.Wrapf(err, "Error running command: %v, Output: %s\n", cmd, string(out))
```

Comment:
Honestly i could be wrong here and im not sure what we do elsewhere during this bootstrap process, but failing during bootstrap is really bad and potentially hard to diagnosis. I wonder if its better in cases like, if we cant get or compare the version, we should just pick a default setup of skips to do, vs failing the entire process?

Reply:
We error out during this process throughout this file. I do agree that something like this comparison might not justify failing the entire process, but it will fail later in the bootstrap process if the we don't skip the correct phases based on the kube version

Reply:
Yea we do fail for commands like this throughout, but we print those logs to journalctl. So if this were to error out, we can still get the logs.